subprojects/launcher/src/main/java/org/gradle/tooling/internal/provider/ProviderConnection.java

```diff
@@ -284,10 +284,6 @@ private Parameters initParams(ProviderOperationParameters operationParameters) {
             if (operationParameters.getGradleUserHomeDir() != null) {
                 layout.setGradleUserHomeDir(operationParameters.getGradleUserHomeDir());
             }
-            Boolean searchUpwards = operationParameters.isSearchUpwards();
```

Comment:
I wonder about this change. The method is only deprecated but effectively ignore here.
Is that correct?

Reply:
That is actually the missing piece for the IJ import and empty `buildSrc` issue. Opening a new PR with this change.